downgrade

```diff
@@ -135,16 +145,18 @@ declare -a terms
 declare -a to_ignore
 declare -a to_install
 declare -a pacman_args
+declare -a version
 
 while [[ -n "$1" ]]; do
   case "$1" in
     --) shift; pacman_args=( "$@" ); break ;;
+    -v) shift; version=( "$1" ); break ;;
```

Comment:
You also shouldn't `break` out of the loop on `-v` or an invocation like `downgrade -v ... -- pacman-options` would never get to the pacman options.